en/docs/learn/fraud-detection.md

```diff
@@ -0,0 +1,422 @@
+authorization to Splunk server and update `http.url` field to that of SPlunk event collector URL as depicted below.
+
+```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <eventPublisher name="IsAnalytics-Publisher-wso2event-AuthenticationData" statistics="disable" trace="disable"
+        xmlns="http://wso2.org/carbon/eventpublisher">
+        <from streamName="org.wso2.is.analytics.stream.OverallAuthentication" version="1.0.0"/>
+        <mapping customMapping="disable" type="json"/>
+        <to eventAdapterType="http">
+            <property name="http.client.method">HttpPost</property>
+            <property name="publishingMode">non-blocking</property>
+            <property name="publishTimeout">0</property>
+            <property name="http.headers">Authorization: Splunk
+                <--created token for Splunk access-->
+            </property>
+            <property name="http.url">http://localhost:8088/services/collector</property>
+        </to>
+    </eventPublisher>
+```
+
+Similarly you can configure any of the available event publishers to be able to communicate with any external SIEM
+over supported event types by WSO2. Once we publish events to these SIEMs, they can use the functionalities of the
+corresponding SIEM to generate security alerts or further processing.
+
+#### Custom Authentication Data Publisher
+
+In WSO2 authentication framework the data publishing module ([AuthnDataPublisherProxy](https://github
+.com/wso2-extensions/identity-data-publisher-authentication/blob/master/components/org.wso2.carbon.identity.data
+.publisher.application.authentication/src/main/java/org/wso2/carbon/identity/data/publisher/
+application/authentication/AuthnDataPublisherProxy.java)) is the component responsible
+for capturing the statistics related to user authentication and session data and invoke the registered data
+publishers to handle the events and publish data to their corresponding destinations. These handlers extend the
+[AbstractEventHandler](https://github.com/wso2/carbon-identity-framework/blob/master/components/identity-event/org
+.wso2.carbon.identity.event/src/main/java/org/wso2/carbon/identity/event/handler/AbstractEventHandler.java) and
+override its handleEvent method. We can easily write a custom data publisher extending the
+AbstractEventHandler and overriding the handleEvent method for the purpose of publishing authentication/session data
+to any third party platform that facilitates fraud detection. When the event publisher triggers an event, there is a
+dedicated service that will transfer this event from the Identity Server to the corresponding destination. This
+service is handled by the Event Adapters. Event adaptors bridge the connection between the data source and the data
+sink. There are several output event adapters supported by the WSO2 products such as **HTTP**, **JMS**, **Web Socket**,
+**SMTP** etc.
+
+Following is a sample code segment on writing a custom data publisher.
```

Comment:
The following code block shows a sample code segment used when writing a custom data publisher.